src/main/java/Response/ResponseConstructor.java

```diff
@@ -0,0 +1,36 @@
+import java.util.HashMap;
+
+public class ResponseConstructor {
+  private String messageBody;
+  private String reasonPhrase;
+  private int statusCode;
+  private HashMap<Integer, String> statusReasonMap = createStatusReasonMap();
+
+  public ResponseConstructor(int statusCode) {
+    this.statusCode = statusCode;
+    this.reasonPhrase = statusReasonMap.get(this.statusCode);
+    this.messageBody = "";
+  }
+
+  public ResponseConstructor(int statusCode, String messageBody) {
+    this.statusCode = statusCode;
+    this.reasonPhrase = statusReasonMap.get(this.statusCode);
+    this.messageBody = messageBody;
+  }
+
+  public Response constructResponse() {
+    return new Response.Builder()
+                       .httpVersion("1.1")
+                       .statusCode(this.statusCode)
+                       .reasonPhrase(this.reasonPhrase)
+                       .messageBody(this.messageBody)
+                       .build();
+  }
+
+  private HashMap<Integer, String> createStatusReasonMap() {
+    HashMap<Integer, String> map = new HashMap<Integer, String>();
+    map.put(200, "OK");
+    map.put(405, "Method Not Allowed");
+    return map;
+  }
+}
```

Comment:
This class might be an unnecessary abstraction...? I understand why you wanted to encapsulate calling Response.Builder, but this might be overkill. The status reason map also seems useful, but why not just create static constants for the codes + reasons like `public static final String OK = "200 OK"` so you don't have to worry about using the codes directly and also finding the appropriate reason?

Reply:
Got it! Is this basically one of the only times where it's acceptable to have public instance variables?

Reply:
Oops. They're static variables. Never mind. Thank you for the suggestion!